### PR TITLE
solution for 68 problem is added

### LINKED ISSUE #3066: [Feature Request]: want to add Add Solution of a problem 68
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

this is solution for 68(hard) problem

### Use Case

this deals with words & maxWidth '  '(extra spaces)

### Benefits

_No response_

### Add ScreenShots

_No response_

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by pallasivasai**
## Description

Fix  #3066 

## Type of PR

- [x] Feature enhancement
- [x] Documentation update

## Screenshots / Videos (if applicable)

Text Justification - LeetCode - Google Chrome 13-07-2024 8_33_17

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have followed the code style guidelines of this project.
- [x] I have checked for any existing open issues that my pull request may address.
- [x] I have ensured that my changes do not break any existing functionality.
- [x] Each contributor is allowed to create a maximum of 4 issues per day. This helps us manage and address issues efficiently.
- [x] I have read the resources for guidance listed below.
- [x] I have followed security best practices in my code changes.

## Additional Context

[Include any additional information or context that might be helpful for reviewers.]

## Resources for Guidance

Please read the following resources before submitting your contribution:

- [x] [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [x] [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [x] [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [x] [Markdown Guide](https://www.markdownguide.org/)
- [x] [MDX Documentation](https://mdxjs.com/docs/)
- [x] [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- [x] [Tabs: Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [x] [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [x] [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

File: dsa-solutions/lc-solutions/0000-0099/0068-Text-Justification.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+```
+
+### Constraints
+
+- 1 <= words.length <= 300
+- 1 <= words[i].length <= 20
+- words[i] consists of only English letters and symbols.
+- 1 <= maxWidth <= 100
+- words[i].length <= maxWidth
+
```
Comment (Ajay-Dhangar) [inline dsa-solutions/lc-solutions/0000-0099/0068-Text-Justification.md:77]: replace `<` to `&lt;`

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
update code based on suggestions

File: dsa-solutions/lc-solutions/0000-0099/0068-Text-Justification.md (hunk 1 above)
Comment (pallasivasai) [reply dsa-solutions/lc-solutions/0000-0099/0068-Text-Justification.md:77]: In which line?

Comment (Ajay-Dhangar) [reply dsa-solutions/lc-solutions/0000-0099/0068-Text-Justification.md:77]:
```md
- 1 <= words.length <= 300
- 1 <= words[i].length <= 20
- words[i] consists of only English letters and symbols.
- 1 <= maxWidth <= 100
- words[i].length <= maxWidth
```

Comment (Ajay-Dhangar) [reply dsa-solutions/lc-solutions/0000-0099/0068-Text-Justification.md:77]: in given block of code you need relace all `<` to `&lt;`

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
update your code

**Review by pallasivasai — COMMENTED** [review]
Will you please atleast what change need to be done and where it is need to be changed mentioned the specific type of change along with the line as well

**Comment by Ajay-Dhangar** [comment]
> Will you please atleast what change need to be done and where it is need to be changed mentioned the specific type of change along with the line as well

wait I solve it

**Review by Ajay-Dhangar — APPROVED** [review]
Done

**Comment by Ajay-Dhangar** [comment]
@pallasivasai, please ensure you read the following resources carefully before submitting your contribution:

## Resources for Guidance

- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- [Tabs: Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

**Comment by pallasivasai** [comment]
Check now @Ajay-Dhangar

**Review by Ajay-Dhangar — APPROVED** [review]
right way like this 

```md
- 1 &lt;= words.length &lt;= 300
- 1 &lt;= words[i].length &lt;= 20
- words[i] consists of only English letters and symbols.
- 1 &lt;= maxWidth &lt;= 100
- words[i].length &lt;= maxWidth
```

**Pull request merged**